typed-racket-doc/typed-racket/scribblings/guide/more.scrbl

```diff
@@ -89,13 +89,21 @@ of @racket[Number]s.
 @racketblock[(lambda ([x : String] (unsyntax @tt["."]) [y : Number #,**]) (apply + y))]
 
 This function has the type @racket[(-> String Number #,** Number)].
+To specify the return type, add a type annotation after the arguments:
+
+@racketblock[(lambda ([x : String] (unsyntax @tt["."]) [y : Number #,**]) : (U Number String) (apply + y))]
```

Comment:
Should we do `(U Number String)` here or just `Number`?

Reply:
Number is boring. I want something other than the default.